crates/test-programs/wasi-tests/src/bin/file_seek_tell.rs

```diff
@@ -25,10 +25,10 @@ unsafe fn test_file_seek_tell(dir_fd: wasi::Fd) {
     assert_eq!(offset, 0, "current offset should be 0");
 
     // Write to file
-    let buf = &[0u8; 100];
+    let data = &[0u8; 100];
     let iov = wasi::Ciovec {
-        buf: buf.as_ptr() as *const _,
-        buf_len: buf.len(),
+        buf: data.as_ptr() as *const _,
+        buf_len: data.len(),
```

Comment:
This is just for consistency; in other tests such as `fd_flags_set.rs` or `file_truncate.rs`, "buffer" (or "buf") is used as a name of mutable buffers, and immutable buffers are called "data" or "contents". As we call `fd_read` below and need to assign a name to the immutable buffer, I thought it might be less confusing to follow the convention.

Reply:
Huh, I didn't know we had a convention for this, but this makes sense! I'll make sure that we keep it that way from now on. Thanks!